Makefile

```
@@ -101,13 +101,13 @@ ifeq ($(ENABLE_WEBHOOKS),true)
 ## v2alpha3, v2alpha4 and v2alpha3 requires allowDangerousTypes=true because they use float32 type
 	cd config/manager && $(KUSTOMIZE) edit add resource webhook_secret.yaml 
 	$(CONTROLLER_GEN) rbac:roleName=$(OPERATOR_CLUSTER_ROLE_NAME) crd:allowDangerousTypes=true webhook paths="./..." output:crd:artifacts:config=config/crd/bases
-	find config -type f -exec sed -i -e '/creationTimestamp/d' {} \;
+	find config -type f -exec ./hack/inplace_sed.sh -e '/creationTimestamp/d' {} \;
```

The `-i.bak` seems working on both `BSD` and `GNU` systems:
```suggestion
	find config -type f -exec sed -i.bak -e '/creationTimestamp/d' {} \; -exec rm {}.bak \;
```